odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -3,6 +3,7 @@
 
 import json
 
+import unittest
```

Comment:
Commit message says:

> For now, the properties fields can be searched only in the top bar

Not very relevant as it's not part of this commit and it comes _after_.
Maybe you could mention that it is meant to be used with the frontend top bar filter on views in a later commit.

In addition, I would reword a bit and mention a few examples of domain that work, and then mention the use cases not supported (poor Bob).

In other words elaborate a bit more, as this is an important feature :)

Reply:
ok, I added more example of what is not supported, and what we should do instead :)